From ec5b90da3daa7c472d7157547d97dc330a812eda Mon Sep 17 00:00:00 2001
From: ripee <ripee.dev@gmail.com>
Date: Sat, 2 Mar 2019 15:53:20 +0100
Subject: [PATCH] Revert "exynos: Use generated kernel headers"

This reverts commit ef5444f6391ecb92fe31c68b540a38a76148ddfe.

fix build errors

libdisplaymodule --> libexynosdisplaymodule
---
 libcec/Android.mk            |  2 ++
 libcsc/Android.mk            |  5 +++--
 libexynosdisplay/Android.mk  |  7 ++++++-
 libexynosutils/Android.mk    |  3 +--
 libfimg4x/Android.mk         |  4 +++-
 libg2d/Android.mk            |  3 +--
 libhdmi/Android.mk           |  8 ++++++--
 libhdmi_dummy/Android.mk     |  6 ++++--
 libhdmi_legacy/Android.mk    |  3 +--
 libhwc/Android.mk            | 13 +++++++++----
 libhwcutils/Android.mk       |  8 ++++++--
 libmpp/Android.mk            |  3 +--
 libv4l2/Android.mk           |  7 +++++--
 libvideocodec/Android.mk     |  9 +++++++--
 libvirtualdisplay/Android.mk |  7 +++++--
 15 files changed, 60 insertions(+), 28 deletions(-)
 mode change 100644 => 100755 libcsc/Android.mk
 mode change 100644 => 100755 libexynosdisplay/Android.mk
 mode change 100644 => 100755 libexynosutils/Android.mk
 mode change 100644 => 100755 libfimg4x/Android.mk
 mode change 100644 => 100755 libg2d/Android.mk
 mode change 100644 => 100755 libhdmi/Android.mk
 mode change 100644 => 100755 libhdmi_dummy/Android.mk
 mode change 100644 => 100755 libhdmi_legacy/Android.mk
 mode change 100644 => 100755 libhwc/Android.mk
 mode change 100644 => 100755 libhwcutils/Android.mk
 mode change 100644 => 100755 libmpp/Android.mk
 mode change 100644 => 100755 libv4l2/Android.mk
 mode change 100644 => 100755 libvideocodec/Android.mk
 mode change 100644 => 100755 libvirtualdisplay/Android.mk

diff --git a/hardware/samsung_slsi-cm/exynos/libcec/Android.mk b/hardware/samsung_slsi-cm/exynos/libcec/Android.mk
index 8b3d462..f101b1d 100644
--- a/hardware/samsung_slsi-cm/exynos/libcec/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libcec/Android.mk
@@ -20,5 +20,7 @@ LOCAL_MODULE_TAGS := eng
 LOCAL_SHARED_LIBRARIES := liblog
 LOCAL_SRC_FILES := libcec.c
 
+LOCAL_CFLAGS += -Wno-unused-function
+
 LOCAL_MODULE := libcec-exynos
 include $(BUILD_SHARED_LIBRARY)
diff --git a/hardware/samsung_slsi-cm/exynos/libcsc/Android.mk b/hardware/samsung_slsi-cm/exynos/libcsc/Android.mk
old mode 100644
new mode 100755
index e13bbde..8a6754c
--- a/hardware/samsung_slsi-cm/exynos/libcsc/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libcsc/Android.mk
@@ -8,12 +8,11 @@ LOCAL_SRC_FILES := \
 	csc.c
 
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	hardware/samsung_slsi-cm/$(TARGET_BOARD_PLATFORM)/include \
 	$(LOCAL_PATH)/../include \
 	$(LOCAL_PATH)/../libexynosutils
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_EXPORT_C_INCLUDE_DIRS := $(LOCAL_PATH)
 
 LOCAL_CFLAGS :=
@@ -43,4 +42,6 @@ endif
 LOCAL_CFLAGS += -DUSE_ION
 LOCAL_SHARED_LIBRARIES += libion_exynos
 
+LOCAL_CFLAGS += -Wno-unused-parameter
+
 include $(BUILD_SHARED_LIBRARY)
diff --git a/hardware/samsung_slsi-cm/exynos/libexynosdisplay/Android.mk b/hardware/samsung_slsi-cm/exynos/libexynosdisplay/Android.mk
old mode 100644
new mode 100755
index c215567..48c0f02
--- a/hardware/samsung_slsi-cm/exynos/libexynosdisplay/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libexynosdisplay/Android.mk
@@ -55,6 +55,7 @@ endif
 
 LOCAL_CFLAGS += -DLOG_TAG=\"display\"
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(LOCAL_PATH)/../libhwc \
 	$(LOCAL_PATH)/../libhwcutils \
@@ -65,7 +66,11 @@ LOCAL_C_INCLUDES := \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhwcmodule \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhwcutilsmodule
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
+LOCAL_CFLAGS += -Wno-unused-parameter
+LOCAL_CFLAGS += -Wno-unused-variable
+LOCAL_CFLAGS += -Wno-sign-compare
+LOCAL_CFLAGS += -Wno-unused-function
+LOCAL_CFLAGS += -Wno-format
 
 LOCAL_SRC_FILES := \
 	ExynosDisplay.cpp \
diff --git a/hardware/samsung_slsi-cm/exynos/libexynosutils/Android.mk b/hardware/samsung_slsi-cm/exynos/libexynosutils/Android.mk
old mode 100644
new mode 100755
index 3c0319f..61e157f
--- a/hardware/samsung_slsi-cm/exynos/libexynosutils/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libexynosutils/Android.mk
@@ -31,10 +31,9 @@ LOCAL_MODULE := libexynosutils
 
 LOCAL_SRC_FILES += exynos_format_v4l2.c
 LOCAL_C_INCLUDES += \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 ifeq ($(BOARD_USES_FIMC),true)
 LOCAL_CFLAGS += -DENABLE_FIMC
 endif
diff --git a/hardware/samsung_slsi-cm/exynos/libfimg4x/Android.mk b/hardware/samsung_slsi-cm/exynos/libfimg4x/Android.mk
old mode 100644
new mode 100755
index 1d534c7..7b7f4cb
--- a/hardware/samsung_slsi-cm/exynos/libfimg4x/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libfimg4x/Android.mk
@@ -25,11 +25,13 @@ LOCAL_SRC_FILES:= \
 	FimgExynos5.cpp
 
 LOCAL_C_INCLUDES += \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	hardware/samsung_slsi-cm/$(TARGET_SOC)/include \
 	hardware/samsung_slsi-cm/$(TARGET_BOARD_PLATFORM)/include
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
+LOCAL_CFLAGS += -Wno-unused-variable
+LOCAL_CFLAGS += -Wno-mismatched-tags
 
 LOCAL_SHARED_LIBRARIES:= liblog libutils libbinder
 
diff --git a/hardware/samsung_slsi-cm/exynos/libg2d/Android.mk b/hardware/samsung_slsi-cm/exynos/libg2d/Android.mk
old mode 100644
new mode 100755
index a115786..e5c462e
--- a/hardware/samsung_slsi-cm/exynos/libg2d/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libg2d/Android.mk
@@ -18,12 +18,11 @@ include $(CLEAR_VARS)
 LOCAL_SHARED_LIBRARIES := liblog libutils libcutils libexynosutils libexynosv4l2
 
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/include \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libexynosutils
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_SRC_FILES := exynos_blender.cpp exynos_blender_obj.cpp libg2d.cpp
 
 LOCAL_MODULE_TAGS := eng
diff --git a/hardware/samsung_slsi-cm/exynos/libhdmi/Android.mk b/hardware/samsung_slsi-cm/exynos/libhdmi/Android.mk
old mode 100644
new mode 100755
index bc17fae..6c8862b
--- a/hardware/samsung_slsi-cm/exynos/libhdmi/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libhdmi/Android.mk
@@ -28,6 +28,7 @@ endif
 LOCAL_CFLAGS += -DLOG_TAG=\"hdmi\"
 
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(LOCAL_PATH)/../libhwcutils \
 	$(LOCAL_PATH)/../libexynosdisplay \
@@ -40,11 +41,14 @@ LOCAL_C_INCLUDES := \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libmpp \
 	$(TOP)/system/core/libsync/include
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_SRC_FILES := \
 	ExynosExternalDisplay.cpp dv_timings.c
 
+LOCAL_CFLAGS += -Wno-header-guard
+LOCAL_CFLAGS += -Wno-missing-field-initializers
+LOCAL_CFLAGS += -Wno-unused-parameter
+LOCAL_CFLAGS += -Wno-unused-variable
+
 LOCAL_MODULE_TAGS := eng
 LOCAL_MODULE := libhdmi
 include $(BUILD_SHARED_LIBRARY)
diff --git a/hardware/samsung_slsi-cm/exynos/libhdmi_dummy/Android.mk b/hardware/samsung_slsi-cm/exynos/libhdmi_dummy/Android.mk
old mode 100644
new mode 100755
index 4937aac..1fb7a0d
--- a/hardware/samsung_slsi-cm/exynos/libhdmi_dummy/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libhdmi_dummy/Android.mk
@@ -37,6 +37,7 @@ ifeq ($(BOARD_USES_FIMC), true)
 endif
 
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(LOCAL_PATH)/../libhwcutils \
 	$(LOCAL_PATH)/../libexynosdisplay \
@@ -49,11 +50,12 @@ LOCAL_C_INCLUDES := \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libmpp \
 	$(TOP)/system/core/libsync/include
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_SRC_FILES := \
 	ExynosExternalDisplay.cpp
 
+LOCAL_CFLAGS += -Wno-unused-parameter
+LOCAL_CFLAGS += -Wno-unused-function
+
 LOCAL_MODULE_TAGS := eng
 LOCAL_MODULE := libhdmi
 include $(BUILD_SHARED_LIBRARY)
diff --git a/hardware/samsung_slsi-cm/exynos/libhdmi_legacy/Android.mk b/hardware/samsung_slsi-cm/exynos/libhdmi_legacy/Android.mk
old mode 100644
new mode 100755
index e8e65db..f632fa8
--- a/hardware/samsung_slsi-cm/exynos/libhdmi_legacy/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libhdmi_legacy/Android.mk
@@ -37,6 +37,7 @@ ifeq ($(BOARD_USES_VIRTUAL_DISPLAY), true)
 endif
 
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(LOCAL_PATH)/../libhwcutils \
 	$(LOCAL_PATH)/../libexynosdisplay \
@@ -49,8 +50,6 @@ LOCAL_C_INCLUDES := \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libmpp \
 	$(TOP)/system/core/libsync/include
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_SRC_FILES := \
 	ExynosExternalDisplay.cpp
 
diff --git a/hardware/samsung_slsi-cm/exynos/libhwc/Android.mk b/hardware/samsung_slsi-cm/exynos/libhwc/Android.mk
old mode 100644
new mode 100755
index 0cb94f1..246d9d8
--- a/hardware/samsung_slsi-cm/exynos/libhwc/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libhwc/Android.mk
@@ -21,7 +21,7 @@ LOCAL_MODULE_RELATIVE_PATH := hw
 LOCAL_SHARED_LIBRARIES := liblog libcutils libEGL libGLESv1_CM libhardware \
 	libhardware_legacy libion_exynos libutils libsync \
 	libexynosv4l2 libMcClient libexynosutils libhwcutils libexynosdisplay libhdmi \
-	libdisplaymodule libhdmimodule libmpp
+	libexynosdisplaymodule libhdmimodule libmpp
 
 ifeq ($(BOARD_DYNAMIC_RECOMPOSITION_DISABLED), true)
 	LOCAL_CFLAGS += -DDYNAMIC_RECOMPOSITION_DISABLED
@@ -93,8 +93,15 @@ ifneq ($(filter exynos7420 exynos7580 exynos7870 exynos7880 exynos8890 exynos889
 endif
 
 LOCAL_CFLAGS += -DLOG_TAG=\"hwcomposer\"
+LOCAL_CFLAGS += -Wno-gnu-designator
+LOCAL_CFLAGS += -Wno-unused-parameter
+LOCAL_CFLAGS += -Wno-unused-variable
+LOCAL_CFLAGS += -Wno-format-extra-args
+LOCAL_CFLAGS += -Wno-unused-function
+LOCAL_CFLAGS += -Wno-format
 
 LOCAL_C_INCLUDES += \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(LOCAL_PATH)/../libhwcutils \
 	$(LOCAL_PATH)/../libexynosdisplay \
@@ -103,14 +110,12 @@ LOCAL_C_INCLUDES += \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libcec \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/include \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhwcmodule \
-	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libdisplaymodule \
+	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libexynosdisplaymodule \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhdmimodule \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhwcutilsmodule \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libmpp \
 	$(TOP)/system/core/libsync/include
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_SRC_FILES := ExynosHWC.cpp
 
 LOCAL_MODULE := hwcomposer.$(TARGET_BOARD_PLATFORM)
diff --git a/hardware/samsung_slsi-cm/exynos/libhwcutils/Android.mk b/hardware/samsung_slsi-cm/exynos/libhwcutils/Android.mk
old mode 100644
new mode 100755
index d13c4b3..7a1443f
--- a/hardware/samsung_slsi-cm/exynos/libhwcutils/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libhwcutils/Android.mk
@@ -53,6 +53,7 @@ endif
 
 LOCAL_CFLAGS += -DLOG_TAG=\"hwcutils\"
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(LOCAL_PATH)/../libhwc \
 	$(LOCAL_PATH)/../libexynosdisplay \
@@ -63,8 +64,6 @@ LOCAL_C_INCLUDES := \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhwcutilsmodule \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libmpp
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 ifeq ($(BOARD_HDMI_INCAPABLE), true)
 LOCAL_C_INCLUDES += $(TOP)/hardware/samsung_slsi-cm/exynos/libhdmi_dummy
 else
@@ -76,6 +75,11 @@ LOCAL_C_INCLUDES += $(TOP)/hardware/samsung_slsi-cm/exynos/libhdmi_legacy
 endif
 endif
 
+LOCAL_CFLAGS += -Wno-format
+LOCAL_CFLAGS += -Wno-mismatched-tags
+LOCAL_CFLAGS += -Wno-unused-parameter
+LOCAL_CFLAGS += -Wno-unused-variable
+
 LOCAL_SRC_FILES += \
 	ExynosHWCUtils.cpp \
 	ExynosMPP.cpp
diff --git a/hardware/samsung_slsi-cm/exynos/libmpp/Android.mk b/hardware/samsung_slsi-cm/exynos/libmpp/Android.mk
old mode 100644
new mode 100755
index dd19c95..807b61c
--- a/hardware/samsung_slsi-cm/exynos/libmpp/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libmpp/Android.mk
@@ -17,6 +17,7 @@ include $(CLEAR_VARS)
 LOCAL_SHARED_LIBRARIES := liblog libutils libcutils libexynosutils libexynosv4l2
 
 LOCAL_C_INCLUDES += \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/include \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libexynosutils \
@@ -24,8 +25,6 @@ LOCAL_C_INCLUDES += \
 	$(TOP)/hardware/samsung_slsi-cm/exynos4/include \
 	$(TOP)/hardware/samsung_slsi-cm/exynos5/include
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_SRC_FILES := MppFactory.cpp
 LOCAL_MODULE_TAGS := eng
 LOCAL_MODULE := libmpp
diff --git a/hardware/samsung_slsi-cm/exynos/libv4l2/Android.mk b/hardware/samsung_slsi-cm/exynos/libv4l2/Android.mk
old mode 100644
new mode 100755
index ad65020..ce2ea96
--- a/hardware/samsung_slsi-cm/exynos/libv4l2/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libv4l2/Android.mk
@@ -22,16 +22,19 @@ LOCAL_SRC_FILES := \
 	exynos_mc.c
 
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libexynosutils
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_SHARED_LIBRARIES := \
 	liblog \
 	libutils \
 	libexynosutils 
 
+LOCAL_CFLAGS += -Wno-unused-variable
+LOCAL_CFLAGS += -Wno-unused-parameter
+LOCAL_CFLAGS += -Wno-unused-function
+
 LOCAL_MODULE := libexynosv4l2
 LOCAL_MODULE_TAGS := eng
 
diff --git a/hardware/samsung_slsi-cm/exynos/libvideocodec/Android.mk b/hardware/samsung_slsi-cm/exynos/libvideocodec/Android.mk
old mode 100644
new mode 100755
index 57bac0e..52ad33b
--- a/hardware/samsung_slsi-cm/exynos/libvideocodec/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libvideocodec/Android.mk
@@ -12,12 +12,11 @@ LOCAL_SRC_FILES := \
 	enc/ExynosVideoEncoder.c
 
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/include \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/include \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_BOARD_PLATFORM)/include
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 ifeq ($(BOARD_USE_KHRONOS_OMX_HEADER), true)
 LOCAL_C_INCLUDES += $(TOP)/hardware/samsung_slsi-cm/openmax/include/khronos
 else
@@ -36,6 +35,12 @@ ifneq ($(filter exynos5422 exynos5430 exynos5433 exynos7420 exynos7580 exynos787
 LOCAL_CFLAGS += -DNEW_API
 endif
 
+LOCAL_CFLAGS += -Wno-unused-parameter
+LOCAL_CFLAGS += -Wno-unused-variable
+LOCAL_CFLAGS += -Wno-unused-function
+LOCAL_CFLAGS += -Wno-unused-label
+LOCAL_CFLAGS += -Wno-format
+
 LOCAL_MODULE := libExynosVideoApi
 LOCAL_MODULE_TAGS := optional
 LOCAL_ARM_MODE := arm
diff --git a/hardware/samsung_slsi-cm/exynos/libvirtualdisplay/Android.mk b/hardware/samsung_slsi-cm/exynos/libvirtualdisplay/Android.mk
old mode 100644
new mode 100755
index 04ae1f4..22a97c1
--- a/hardware/samsung_slsi-cm/exynos/libvirtualdisplay/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libvirtualdisplay/Android.mk
@@ -24,8 +24,13 @@ endif
 
 LOCAL_CFLAGS += -DUSES_VIRTUAL_DISPLAY
 LOCAL_CFLAGS += -DLOG_TAG=\"virtual\"
+LOCAL_CFLAGS += -Wno-format
+LOCAL_CFLAGS += -Wno-mismatched-tags
+LOCAL_CFLAGS += -Wno-sign-compare
+LOCAL_CFLAGS += -Wno-unused-parameter
 
 LOCAL_C_INCLUDES := \
+	$(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include \
 	$(LOCAL_PATH)/../include \
 	$(LOCAL_PATH)/../libhwcutils \
 	$(LOCAL_PATH)/../libexynosdisplay \
@@ -37,8 +42,6 @@ LOCAL_C_INCLUDES := \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhwcutilsmodule \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libmpp
 
-LOCAL_HEADER_LIBRARIES := generated_kernel_headers
-
 LOCAL_SRC_FILES := \
 	ExynosVirtualDisplay.cpp
 
-- 
2.17.1

